### PR TITLE
license compliance

### PR DESCRIPTION
**Pull request by azakus**

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -8,6 +8,15 @@
 -->
 
 <!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
```
Comment (notwaldorf) [inline demo/index.html]: but the license was just above it? hmmm

Comment (cdata) [inline demo/index.html]: derp.

**Comment by notwaldorf** [comment]
Soooooo should I close this?

**Comment by azakus** [comment]
License should be after doctype I think

**Comment by notwaldorf** [comment]
Oh this looks ok now. Thanks!

**Pull request merged**